src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -474,6 +473,16 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator, IRBlobSe
         private async Task RunWorker(CancellationToken ct) {
             TaskUtilities.AssertIsOnBackgroundThread();
 
+            // Spin until the worker task is registered.
+            while (_runTask == null) {
+                await Task.Yield();
```

Comment:
Ok

---
In reply to: [75006462](https://github.com/Microsoft/RTVS/pull/2173#discussion_r75006462) [](ancestors = 75006462,74950109)